packages/docs/src/layouts/default.vue

```diff
@@ -230,6 +230,13 @@ html {
 
     overflow-y: auto;
     overflow-x: hidden;
+
+    @media(max-width:670px){
+      & > :last-child {
+        padding:0em !important;
```

Comment:
```suggestion
        padding: 0rem;
```

You don't need an `!important` if you will use more priority selector (`.layout.gutter--xl` instead `:last-child`)